src/controls/GridControl/_grid.scss

```diff
@@ -18,16 +28,14 @@
 
 .mapbox-control-grid-selector-modal {
   &-root {
-    width: 100%;
-    height: 100%;
+    @include size(100%, 100%);
     display: none;
     position: fixed;
     z-index: 1;
     left: 0;
```

Comment:
Perhaps we can create an 'absolutePosition' mixin too, to specify the top, left, right and bottom rules

Reply:
i'm not sure what rules to add, theres just one case where we have rules but it's according to the button placement (mapTilesControlPosition)